public/app/features/dashboard-scene/settings/version-history/HistorySrv.test.ts

```diff
@@ -34,7 +34,7 @@ describe('historySrv', () => {
   });
 
   describe('getHistoryList', () => {
-    it('should return a versions array for the given dashboard id', () => {
+    it('should return a versions array for the given dashboard id', async () => {
```

Comment:
small nit: if you're marking the test as `async`, probably you can use `await` instead of promises 😄